src/Tribe/Utils/Post_Root_Pool.php

```diff
@@ -88,7 +88,7 @@ public function get_pool_transient_name() {
 	 * @return string
 	 */
 	protected function uc_first_letter( $string ) {
-		return is_numeric( $string ) ? $string : strtoupper( $string[0] );
+		return is_numeric( $string ) ? $string : mb_strtoupper( $string[0], mb_detect_encoding( $string[0] ) );
```

Comment:
Any reason not to use `ucfirst` instead?

Reply:
Yeah, `ucfirst()` is locale-specific? Which _might_ not work well if the locale is US and the Event has a German title? (won't capitalize ä for example)

Reply:
Got it I think you're right as it might use the current locale.